proptest-regressions/config/mod.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
Is this new? Not seeing where this comes from.

Reply:
Ah darn it, that sneaked in from another branch..